tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -15,6 +15,12 @@ platform_tests/fwutil/test_fwutil.py::test_fwutil_auto:
   skip:
     reason: "Command not yet merged into sonic-utilites"
 
+everflow/test_everflow_testbed.py::TestEverflowV4IngressAclIngressMirror::test_everflow_dscp_with_policer:
+  skip:
```

Comment:
Can we do xfail with strict being true instead? This way we know there is a feature gap.

Reply:
@yxieca : Made the changes as per your comments